src/main/kotlin/lotto/number/LottoNumber.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.number
+
+data class LottoNumber(
+    val number: Int,
+) {
+    init {
+        require(number in MIN_NUMBER..MAX_NUMBER) { "로또 번호는 ${MIN_NUMBER}와 ${MAX_NUMBER}사이여야 합니다." }
+    }
+
+    companion object {
+        private const val COUNT = 6
+        const val MIN_NUMBER = 1
+        const val MAX_NUMBER = 45
+        val RANGE: List<LottoNumber> = (MIN_NUMBER..MAX_NUMBER).map { LottoNumber(it) }.toList()
```

Comment:
다음과 같이 작성해볼 수 있을 것 같아요!
```suggestion
        val RANGE: List<LottoNumber> = (MIN_NUMBER..MAX_NUMBER)
            .map(::LottoNumber)
            .toList()
```

- https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
- https://kotlinlang.org/docs/reflection.html#callable-references

Reply:
컨벤션 맞춰서 수정했어요! 역시 훨씬 코드 읽기 편한것 같아요 :smile: